omr/CMakeLists.txt

```diff
@@ -59,8 +59,6 @@ if(OMR_GC)
 	target_link_libraries(omrcore
 		PRIVATE
 			${OMR_CORE_GLUE_TARGET}
-			omrtrace
```

Comment:
I thought the OMR core code was responsible for calling the GC startup APIs and also used tracepoints. Doesn't that mean that it needs to link the gc and trace libraries?

Reply:
It means the end consumer needs to link against those (if they actually end up calling said functions). However the issue is that almost everyone links against omrcore, thus making omrcore depend on anything quickly introduces cycles